Tests/StripeTests/Test+Droplet.swift

```diff
@@ -15,7 +15,7 @@ extension XCTestCase {
     func makeDroplet() throws -> Droplet {
         let config = Config([
             "stripe": [
-                "apiKey": "sk_test_O4dmN96uthPP7tvR8dNkPWZg" // Add your own API Key for tests
+                "apiKey": "sk_test_Wxn8UzIs9dkIR4qJYAtHhvY8" // Add your own API Key for tests
```

Comment:
Did the test API key for the test account not work?

Reply:
I changed it. I was having XCODE problems where running the tests wouldn't work unless I added breakpoints and waited until one finished. I thought it was a api key error at first so I rolled a new one.

Reply:
Oh okay,  gotcha